.github/workflows/e2e.yml

```diff
@@ -77,25 +69,8 @@ jobs:
           echo "After removing files:"
           df -h
 
-      - name: Get Control Plane endpoint IP
-        id: get-control-plane-endpoint-ip
-        if: inputs.provider == 'Nutanix'
-        run: |
-          export CONTROL_PLANE_ENDPOINT_RANGE_START="${{ vars.NUTANIX_CONTROL_PLANE_ENDPOINT_RANGE_START }}"
-          export CONTROL_PLANE_ENDPOINT_RANGE_END="${{ vars.NUTANIX_CONTROL_PLANE_ENDPOINT_RANGE_END }}"
-          control_plane_endpoint_ip="$(devbox run -- make nutanix-cp-endpoint-ip)"
```

Comment:
we should remove `nutanix-cp-endpoint-ip:` from makefile

Reply:
Good catch!